### PR TITLE
Add disableInteraction layer with customizable className

### PR DESCRIPTION
**Pull request by msliva**
And in same time I added custom class Name for this element because in my case we want to have border around highlighted element

### CODE REVIEW

**Comment by elrumordelaluz** [comment]
Thanks @msliva for pointing this.

For how I designed the idea of `reactour`, this is in reallity a _feature_, not a problem. I mean, for me was an important requirement to allow the _user_ to interact with the highlighted `elem`. in fact, there are many calculations that the _Tour_ do with _Masks_ to let the user interact. It would've been easier to not allow, since we could add a generic _Mask_ covering all the `window`.

Appreciate your contribution, because days ago I was thinking just in the fact that obviously this _feature_ is not needed in all scenarios (probably your _use-case_ included). 

I think that the _Tour_ needs a `prop` something like `disableInteraction` for those cases, where the _Tour_ will add the `ElementMask` you are proposing.

Also love the idea to made it customizable through `className` and so on…

**Comment by msliva** [comment]
Hi @elrumordelaluz I have added `disableInteraction` I hope that it is how you imagine

Best Michal

**Comment by elrumordelaluz** [comment]
@msliva were perfect those changes! 
I'm sorry did some review but didn't commit the request. Let me know if there is something you want to talk about. Otherwise, if you are agree with the lil changes, when ready will merge it.

Thanks again.

File: README.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ const steps = [
 | `badgeContent` | Function to customize Badge content `(current, total) => {} ` | `func` |  |  |
 | `className` | Custom class to add to the **helper** | `string` |  |  |
 | `closeWithMask` | Close clicking the **mask** | `bool` | `true` |  |
+| `elementClassName` | Custom class name for element which is overlaid target element | `string` |  |  |
```
[the remark this answers is not in the document]
Comment (msliva) [reply README.md]: I have renamed elementClassName to highlightedMaskClassName. And I am planning this className use for something
screen shot 2017-09-18 at 12 07 47
 like this

**Comment by msliva** [comment]
Am I correct, that rights for merge this pull request has only you? Because I don't see here button for merge :) For me it is first time what I am contributing on github.

**Comment by elrumordelaluz** [comment]
>Am I correct, that rights for merge this pull request has only you

Yes, don't worry will merge soon, I'm sorry was busy.

> For me it is first time what I am contributing on github.
screen shot 2017-09-18 at 15 43 04

yes, I saw the GH badge and love to let you do through `reactour`!

I am curious if [this image](https://github.com/elrumordelaluz/reactour/pull/15/commits/603bd2feeea0674d0fef097b90e8c3d65da17d44#r139382243) is `reactour`?

**Pull request merged**